### PR TITLE
[12.0][MIG] sale_global_discount: Migration to 12.0

### PR DESCRIPTION
**Pull request by pedrobaeza**
* Standard procedure
* Code adaptation
* README regeneration
* Tests adaptation

@Tecnativa TT24295

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]

File: sale_global_discount/README.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Settings > Parameters > Global Discounts*
```
Comment (HaraldPanten) [inline sale_global_discount/README.rst]:
```suggestion
#. Go to *Settings > Technical > Parameters > Global Discounts*
```

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍 . Just minor comment

File: sale_global_discount/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply sale_global_discount/README.rst]: Fixed

**Pull request merged**